samples/guestbook/inject_gcp.go

```diff
@@ -71,13 +72,13 @@ func gcpSQLParams(id gcp.ProjectID, flags *cliFlags) *cloudmysql.Params {
 
 // gcpMOTDVar is a Wire provider function that returns the Message of the Day
 // variable from Runtime Configurator.
-func gcpMOTDVar(ctx context.Context, client *runtimeconfigurator.Client, project gcp.ProjectID, flags *cliFlags) (*runtimevar.Variable, func(), error) {
+func gcpMOTDVar(ctx context.Context, client pb.RuntimeConfigManagerClient, project gcp.ProjectID, flags *cliFlags) (*runtimevar.Variable, func(), error) {
```

Comment:
I'm generally in favor of reducing the middle layer, or wrapper. One minor concern is that user may need to figure out want's inside package "google.golang.org/genproto/googleapis/cloud/runtimeconfig/v1beta1". Though this can be trivial and only affect users using all default options we provide. I would imagine in real applications user would  do that anyway to create their own custom clients?

Reply:
Probably. And this is not any worse than before; before this PR, they needed to create one of these to create the `Client`, now they need to create one to create the `Variable`.